mothernet/models/mothernet_additive.py

```diff
@@ -126,19 +126,26 @@ def forward(self, src, single_eval_pos=None):
 
 def bin_data(data, n_bins, single_eval_pos=None):
     # data is samples x batch x features
-    # FIXME treat NaN as separate bin
-    data_nona = torch.nan_to_num(data, nan=0)
-    quantiles = torch.arange(n_bins + 1, device=data.device) / n_bins
+    quantiles = torch.arange(n_bins - 1, device=data.device) / (n_bins - 2)
```

Comment:
you subtracted two from ``n_bins``, I assume you wanted to subtract one, right? I figure that's where the missing bin is coming from?

Reply:
I don't understand why that would mean you need to subtract two. For the actual quantiles, you just want to use the same code with one less bin, right?